static/js/vue-cdr-access/src/components/browseSearch.vue

```diff
@@ -78,16 +78,16 @@ Search form component displayed on full record pages, allowing keyword searches
 
         methods: {
             getResults() {
-                let update_params = { anywhere: encodeURIComponent(this.search_query) };
+                let update_params = { anywhere: encodeURIComponent(this.search_query), start: 0 };
                 this.routeWithParams(this.urlParams(update_params, 'displayRecords'));
             },
 
             clearSearch() {
-                this.routeWithParams(this.removeQueryParameters(this.allPossibleSearchParameters));
+                this.routeWithParams(this.removeQueryParameters(this.allPossibleSearchParameters, true));
             },
 
             clearAllFacets() {
-                this.routeWithParams(this.removeQueryParameters(this.possibleFacetFields));
```

Comment:
I don't think this is working, if I clear all facets it stays on the current page for me on dcr-test.

For the sake of readability, I think it might be good to break this line up into 2 or 3 lines with good variable names, or good comments, since its a decent number of steps

Reply:
There are a two other places that we should reset the page number:
* Clicking "Show only works"
* Changing the sort (this one doesn't break, but is confusing)